tools/goseq_1_22_0/goseq.xml

```diff
@@ -1,62 +1,128 @@
-<tool id="goseq" name="goseq gene ontology analyser" version="0.1.0">
-    <description />
-    <requirements>
-        <requirement type="package" version="3.2.1">R</requirement>
-        <requirement type="package" version="1.22.0">goseq</requirement>
-    </requirements>
+<tool id="goseq" name="goseq" version="0.2.2">
+    <description>tests for overrepresented gene categories</description>
+    <macros>
+        <import>go_macros.xml</import>
```

Comment:
Well, maybe using the macros fits with the guide lines... But in this case this is the same number of code lines and does not really improve the readability.